prime-router/settings/organizations.yml

```diff
@@ -1348,6 +1348,207 @@
   jurisdiction: STATE
   stateCode: PA
   receivers:
+    - name: elr-chester-hl7
+      organizationName: pa-phd
+      topic: covid-19
+      customerStatus: active
+      jurisdictionalFilter: [ "matches(ordering_facility_state, PA)", "matches(ordering_facility_county, Chester, Delaware)" ]
+      translation:
+        type: HL7
+        useBatchHeaders: true
+        nameFormat: chester_hl7
+        receivingApplicationName: PA-ELR
+        receivingFacilityName: PADOH
+        suppressHl7Fields: PID-13-1,ORC-23-1,OBR-16
+        replaceValue:
+          ORC-21-2: L
+        valueSetOverrides:
+          hl70189:
+            name: hl70189
+            system: HL7
+            version: 2.9
+            reference: HL7 guidance for ethnicity, modified for PA standards
+            values:
+              - code: H
+                display: Hispanic or Latino
+              - code: NH
+                replaces: N
+                display: Non Hispanic or Latino
+              - code: U
+                display: Unknown
+      transport:
+        type: SOAP
```

Comment:
@RickHawesUSDS I think up to and including the envelope, it can be generic. The customization comes in when creating the POJO's, which is obviously very custom. We could certainly keep the envelope and the logic for submitting to a SOAP service fairly generic, as long as the POJO's convert themselves to XML. I do think that there's a case where we will need to extend out the SOAP transport (for PHINMS for example) and that that will require some additional work.

Reply:
Ok.